content/docs/components-and-props.md

```diff
@@ -111,11 +109,6 @@ function App() {
     </div>
   );
 }
-
```

Comment:
We decided to leave this render intentionally? 

preview: https://reactjs-org-7ht1fcs9w-fbopensource.vercel.app/docs/components-and-props.html#composing-components

docs: https://reactjs.org/docs/components-and-props.html#composing-components

Reply:
Yeah, this code isn't important/needed for the example